Tuist/Interfaces/SwiftUI/Project/Podfile

```diff
@@ -12,9 +12,12 @@ end
 target '{PROJECT_NAME}' do
   # UI
   pod 'Kingfisher'
+  pod 'FlowStacks'
 
   # Backend
-  pod 'Alamofire'
+  # Network
+  pod 'Moya/Combine'
+  pod 'JSONAPIMapper', :git => 'https://github.com/nimblehq/JSONMapper', :branch => 'main'
```

Comment:
We have tag, so please use tag instead of branch

Reply:
solved in [232caf4](https://github.com/nimblehq/ios-templates/pull/506/commits/232caf454645aac9cc6485e5a747b4e1f1ca0fd9)